web/packages/plugins/apps/analysis-platform/src/modules/Analysis/AnalysisOverview.tsx

```diff
@@ -2,7 +2,7 @@ import { AuthContext, DmtSettings } from '@dmt/common'
 import React, { ReactNode, useContext } from 'react'
 import AnalysisTable from './components/AnalysisTable'
 import { Link, useLocation } from 'react-router-dom'
-import { hasExpertRole } from '../../utils/auth'
+import { hasExpertRole } from '@dmt/analysis-platform'
```

Comment:
Should a package not use relative references for internal imports?
Whats the pros/cons of doing it this way?